analysis/report_templates/default.html

```diff
@@ -232,17 +232,51 @@ <h5 class="center-align">Mann-Whitney U test</h4>
                 </li>
             </ul>
 
-            {% if coverage_report %}
+            {% if detailed_coverage_report %}
+            <ul class="collapsible">
+                <li>
+                    <div class="collapsible-header">
+                        Differential matrix
+                    </div>
+                    <div class="collapsible-body">
+
+                        <div class="row">
+                            <div class="col s6 offset-s3">
+                                <h5 class="center-align">Ranking by unique edges covered</h4>
+                                    <img class="responsive-img materialboxed"
+                                         src="{{ benchmark.unique_region_ranking_plot }}">
+                                    The table shows the unique edges coverage rank of each fuzzer
+                                    on this benchmark. Unique edges are edges only covered by one
```

Comment:
Consider just saying something like:

`Each bar shows the total number of edges found by a given fuzzer. The colored area shows the number of unique edges (i.e., edges that were not covered by any other fuzzers).`